unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,96 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import time
+import pyrealsense2 as rs
+
+from rspy import test
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :sensor: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def set_and_verify_timestamp_domain(sensor, global_time_enabled: bool):
+    """
+    Perform sensor (depth or color) test according given global time
+    :sensor: depth or color sensor in device
+    :global_time_enabled bool: True - timestamp is enabled otherwise false
+    """
+    global frame_queue
+
+    try:
+        sensor.set_option(rs.option.global_time_enabled, 1 if global_time_enabled else 0)
+        time.sleep(0.7)
+        frame = frame_queue.wait_for_frame()
+
+        if frame_queue is None or frame_queue.size() == 0:
+            test.fail()
+
+        expected_ts_domain = rs.timestamp_domain.global_time if global_time_enabled else \
+            rs.timestamp_domain.hardware_clock
+
+        test.check_equal(sensor.get_option(rs.option.global_time_enabled), 1 if global_time_enabled else 0)
+        test.check_equal(frame.get_frame_timestamp_domain(), expected_ts_domain)
+
+    except Exception as exc:
+        print(str(exc))
+        test.fail()
+
+
+depth_sensor = None
+color_sensor = None
+
+try:
+    frame_queue = rs.frame_queue(capacity=5, keep_frames=False)
```

Comment:
Can we try to use `capacity=1` and reduce the sleep time?

Reply:
Capacity was changed to 1.
All tests PASS without `time.sleep()`. I suppose that it happened because `wait_for_frame()` waiting to.

Here logs:
> Check setting global time domain: depth sensor - timestamp domain is OFF
> <pyrealsense2.frame Z16 #1 @1386441.822000> timestamp_domain.hardware_clock
> Test passed
> 
> ___
> Check setting global time domain: depth sensor - timestamp domain is ON
> <pyrealsense2.frame Z16 #2 @1664697594170.816650> timestamp_domain.global_time
> Test passed
> 
> ___
> Check setting global time domain: color sensor - timestamp domain is OFF
> <pyrealsense2.frame RGB8 #1 @1387789.425000> timestamp_domain.hardware_clock
> Test passed
> 
> ___
> Check setting global time domain: color sensor - timestamp domain is ON
> <pyrealsense2.frame RGB8 #2 @1664697595546.604248> timestamp_domain.global_time
> Test passed
> 
> ___
> All tests passed (8 assertions in 4 test cases)

What are you thing we need "sleep" function or we can go ahead without it?

Reply:
Updated to: `time.sleep(0.3)`